jdbc-adapter/README.md

```diff
@@ -137,9 +144,11 @@ You have to make sure that Exasol can connect to the host running the udf_debug.
 
 
 ## Frequent Issues
-* **Error: No suitable driver found for jdbc...**: The jdbc driver class was not discovered automatically. Either you have to add a META-INF/services/java.sql.Driver file with the classname to your jar, or you have to load the driver manually (see JdbcMetadataReader.readRemoteMetadata()).
+* **Error: No suitable driver found for JDBC...**: The JDBC driver class was not discovered automatically. Either you have to add a META-INF/services/java.sql.Driver file with the classname to your jar, or you have to load the driver manually (see JdbcMetadataReader.readRemoteMetadata()).
 See https://docs.oracle.com/javase/7/docs/api/java/sql/DriverManager.html
-
+* **Very slow execution of queries with SCRIPT_OUTPUT_ADDRESS**: If `SCRIPT_OUTPUT_ADDRESS` is set as explained in the [debugging section](#debugging), verify that a service is actually listening at that address. Otherwise, if Exasol can not establish a connection, repeated connection attempts can be the cause for slowdowns.
+* **Very slow execution of queries**: Depending on which JDK version Exasol uses to execute Java user-defined functions, a blocking randomness source may be used by default. Especially cryptographic operations do not complete until the operating system has collected a sufficient amount of entropy. This problem seems to occur most often when Exasol is run in an isolated environment, e.g., a virtual machine or a container. A solution is to use a non-blocking randomness source. 
+  To do so, log in to EXAOperation and shutdown the database. Append `-etlJdbcJavaEnv -Djava.security.egd=/dev/./urandom` to the "Extra Database Parameters" input field and power the database on again.
```

Comment:
This will fix the slowness of the IMPORT FROM JDBC, however, there are also JDBC metadata calls directly from the Adapter script, which will be executed in a nested container environment.
If this is the case, the %env (or javaoptions, forgot the details) would need to be set in the adapter script.
Did you also do this to fix the slowness issues?

Reply:
I'm not so sure about this case. Before submitting this PR I did some quick tests with setting the Java options in both EXAOperations and the adapter script (using `%env` variables), setting them in only one of the two, and not setting them at all. Here are the results running the integration tests two times for each combination (`y` meaning "Java option set", `n` meaning "option not set"):
```
EXAOperation y, Adapter Script n: 06:32 min / 6:32 min
EXAOperation y, Adapter Script y: 06:30 min / 6:46 min
EXAOperation n, Adapter Script y: 29:47 min / 31:09 min
EXAOperation n, Adapter Script n: 14:35 min / 14:50 min
```
The results are pretty surprising (especially comparing the last two cases) but that's what I based my recommendation on.